src/bundles/pix_n_flix/functions.ts

```diff
@@ -224,7 +236,11 @@ function startVideo(): void {
  */
 function stopVideo(): void {
   if (!videoIsPlaying) return;
-  pauseVideoElement();
+  if (inputFeed === 'camera') {
```

Comment:
Just something about the if-else style I noticed, sometimes a single line is used (eg changing to `if ( ... ) return;` at L324), which could be applied here

Reply:
Seems like we can't make it a single line statement here since we still need to call window.cancelAnimationFrame in line 244 regardless of what inputFeed is.

Reply:
![image](https://user-images.githubusercontent.com/44526554/168632100-2a95bb3a-f14f-4524-b1e1-be10df16c10a.png)

Here is the change I was referencing, going from an if block to a single-line if. I just noticed that the same could be applied elsewhere, if moving towards a particular style.

```js
if (inputFeed === 'camera') {
  videoIsPlaying = false;
} else {
  pauseVideoElement();
}
window.cancelAnimationFrame(requestId);
```

VS

```js
if (inputFeed === 'camera') videoIsPlaying = false;  
else pauseVideoElement();

window.cancelAnimationFrame(requestId);
```

Reply:
Ohhhh okay